src/app/Player.js

```diff
@@ -11,3 +11,19 @@ export class Player {
     onQuestionAnswered(chosenAnswer);
   }
 }
+
+export const computerMind = {
+  question: undefined,
+  setQuestion(question) {
+    computerMind.question = question;
+  },
+  tryToAnswer() {
+    return computerMind.question.answers[randomAnswer(question.answers)];
+  },
+};
+
+const randomAnswer = (questionAnswers) => {
```

Comment:
Za jakąś godzinkę wrócę do domu i przeanalizuję, jak to działa, bo coś mi tutaj śmierdzi. Co ta funkcja zwraca? I jak to może działać z tryToAnswer.

Reply:
Hehehe, 2 godziny kombinowałem ;)